doc/source/new-project-guide/level-5-tutorial.rst

```diff
@@ -218,11 +218,17 @@ We require that each PR includes a news item as a ``<branch-name>.rst`` file und
         git checkout skpkg-migration
         git pull origin skpkg-migration
 
-#. Make a copy of ``news/TEMPLATE.rst`` and rename it to ``news/<branch-name>.rst``.
+#. Make a copy of ``news/TEMPLATE.rst`` and rename to ``news/<branch-name>.rst``. We receommend use the following command to streamline this process.
```

Comment:
I don't think I recommend this.  It is a lot of typing and much more work than doing it in pycharm ormanually typing the branch name etc..  I suggest to remove this.  Just leave the (optional) think below

Reply:
Done - removed this "recommend"